securedrop_client/gui/widgets.py

```diff
@@ -2316,9 +2331,17 @@ def _set_file_name(self) -> None:
             self.horizontal_line.hide()
             self.spacer.show()
 
+    @pyqtSlot(state.FileId)
+    def _on_file_download_started(self, id: state.FileId) -> None:
+        if str(id) == self.uuid:
+            logger.debug(f"Starting animation: {id}")
+            self.downloading = True
+            QTimer.singleShot(300, self.start_button_animation)
```

Comment:
Use `1` for the delay. There is no point in waiting before the animation **starts**.

Reply:
Done